From: Emmanuel Arias <eamanu@yaerobi.com>
Date: Wed, 4 Nov 2020 16:31:06 -0300
Subject: Remove cmake extension

Remove cmake extension because is no package on Debian yet. This patch
must be removed after sphinxcontrib-cmake is package on Debian.
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 094ed23..14ce159 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -41,7 +41,7 @@ import skbuild
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
-    'cmake',
+    # 'cmake',
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
     'sphinx_issues'
